src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -204,6 +204,69 @@ mocha.describe('readings API', () => {
                 // Add BG11 here
 
                 // Add BG12 here
+                mocha.it('BG12: 1 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as kg of CO2', async () =>{
```

Comment:
Could you please remove the comment on the previous line abouty add BG12 here since it is now done.

Reply:
Done!